main.go

```diff
@@ -0,0 +1,9 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println("Hello World!") // nolint:forbidigo
```

Comment:
Added in purpose so it can be used as an example of how to skip linting for some particular lines. In this case, most of the `fmt.Print*` calls are forbidden since most of the times are developers adding "debug" lines in their code.